src/status_im/ui/screens/wallet/request/styles.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.ui.screens.wallet.request.styles
+  (:require [status-im.components.styles :as styles]))
+
+(def main-container
+  {:flex 1})
+
+(def network-label
+  {:margin-top 27})
+
+(def network-container
+  {:flex        1
+   :align-items :center})
+
+(def qr-container
+  {:margin-top 16})
+
+(def choose-wallet-container
+  {:margin-top        27
+   :margin-horizontal 15})
+
+(def amount-container
+  {:margin-top        16
+   :margin-horizontal 15
+   :flex-direction    :row})
+
+(def choose-currency-container
+  {:margin-left 8})
+
+(def choose-currency
+  {:width 116})
+
+(def separator
+  {:height            1
+   :margin-horizontal 15
+   :background-color  styles/color-white-transparent-1
+   :margin-top        16})
+
+(def buttons-container
+  {:margin-vertical    15
+   :padding-horizontal 12
+   :flex-direction     :row
+   :align-items        :center})
+
+(def share-icon-container
+  {:margin-right 8})
+
+(def forward-icon-container
+  {:margin-left 8})
+
+(def button-text
```

Comment:
Can those one be shared? Should all buttons have same style?

Reply:
don't think so, it's difficult tot say

Reply:
Have you looked at `components.action-button` ? Those are used in the rest of wallet screens and look similar.

Reply:
this code for these buttons
![image](https://user-images.githubusercontent.com/11790366/30365894-e817a4b8-9871-11e7-91d3-06a20a716162.png)
not sure this the same as `components.button`

Reply:
Ok lookks like more advanced indeed.